HISTORY.md

```diff
@@ -1,3 +1,197 @@
+# TileDB v2.9.0 Release Notes
+
+## Disk Format
+
+* Update on-disk format because of the new available compressor for Dictionary-encoding of strings [#3042](https://github.com/TileDB-Inc/TileDB/pull/3042)
+
+## New features
+
+* RLE compression support for var-length string dimensions [#2938](https://github.com/TileDB-Inc/TileDB/pull/2938)
```

Comment:
I believe this was done in 2.8 , I verified it's included in New features 2.8.0 in this file.